backend/controller/observability/async_calls.go

```diff
@@ -64,11 +66,22 @@ func initAsyncCallMetrics() (*AsyncCallMetrics, error) {
 		result.completed = noop.Int64Counter{}
 	}
 
+	counterName = fmt.Sprintf("%s.ms_to_complete", asyncCallMeterName)
+	if result.msToComplete, err = result.meter.Int64Histogram(
+		counterName,
+		metric.WithUnit("ms"),
+		metric.WithDescription("duration in ms to complete an async call, from the earliest time it was scheduled to execute")); err != nil {
+		errs = errors.Join(errs, fmt.Errorf("%q histogram init failed; falling back to noop: %w", counterName, err))
```

Comment:
I asked in another PR, but why are we returning joined errors rather than just early returning the first error?

Reply:
Wes and I were just talking offline about this and decided we should refactor this to return the first error along with some other minor stylistic things, so I'll update to the new approach in this PR after standup